exercise/models.py

```diff
@@ -11,10 +11,12 @@
 
 EXERCISE_EMAIL_PHISH = 0
 EXERCISE_EMAIL_REGULAR = 1
+EXERCISE_EMAIL_ETRAY = 2
 
 EXERCISE_PHISH_TYPES = (
     (EXERCISE_EMAIL_PHISH, "phishing"),
     (EXERCISE_EMAIL_REGULAR, "regular"),
+    (EXERCISE_EMAIL_ETRAY, "etray"),
```

Comment:
why are we adding these back? wasn't there a ticket to remove these?

Reply:
I've added because somewhere in the middle this was required and i thought somebody took it out by mistake. Ill take a look.

Reply:
Removed.